src/Compilers/Core/Portable/Syntax/SyntaxAnnotation.cs

```diff
@@ -18,7 +18,7 @@ public sealed class SyntaxAnnotation : IObjectWritable, IObjectReadable, IEquata
         /// <summary>
         /// A predefined syntax annotation that indicates whether the syntax element has elastic trivia.
         /// </summary>
-        public static readonly SyntaxAnnotation ElasticAnnotation = new SyntaxAnnotation();
+        public static SyntaxAnnotation ElasticAnnotation { get; } = new SyntaxAnnotation();
```

Comment:
public static readonly fields of an immutable type are explicitly mentioned as OK in the Framework Design Guidelines. 

The field effectively promises two things: (1) identity guarantee, (2) initialization is done in cctor.

The second one is an implementation detail I prefer to be free to change and the first is fine if simply documented and not expressed in the type system IMHO. 

There are also competing precedents: see Encoding.UTF8 vs. String.Empty.  

I'd say it's up to you: we have no guidelines saying this can't be a field, nor any saying this can't be a property.